src/server/routes/entity/creator.js

```diff
@@ -95,17 +96,13 @@ router.get(
 			'Creator'
 		);
 
-		const props = generateProps(req, res, {
-			entityType: 'creator',
-			genderOptions: res.locals.genders,
-			heading: 'Create Creator',
-			identifierTypes: filteredIdentifierTypes,
-			initialState: {},
-			languageOptions: res.locals.languages,
-			requiresJS: true,
-			subheading: 'Add a new Creator to BookBrainz',
-			submissionUrl: '/creator/create/handler'
-		});
+		const props = generateEntityProps(
+			'creator', 'create', req, res, {
+				genderOptions: res.locals.genders,
+				identifierTypes: filteredIdentifierTypes,
+				submissionUrl: '/creator/create/handler'
```

Comment:
Maybe we can set a default for this too - I'd suggest passing in a BBID or null value to the generateEntityProps function, and then using that to determine whether it's a create or edit action. Then you can also use the BBID for the submission URL generation (if not null).

Reply:
I can use `res.locals.entity` to determine if this is a create or edit action and use it accordingly.